package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "virtual-keyboard",
+  "version": "1.0.0",
+  "description": "��#\u0000 \u0000v\u0000i\u0000r\u0000t\u0000u\u0000a\u0000l\u0000-\u0000k\u0000e\u0000y\u0000b\u0000o\u0000a\u0000r\u0000d\u0000\r\u0000 \u0000",
+  "main": "index.js",
+  "scripts": {
+    "test": "npm run lint",
+    "lint": "node node_modules/eslint/bin/eslint.js ./modules"
```

Comment:
usually once you install a package like
`npm i --save-dev eslint`
you can use it in `scripts` like
`eslint src`

Reply:
Understood, did not had a `eslint-cli installed` , so `eslint` coomand was not recognizable.
Fixed!3